packages/material-ui-icons/package.json

```diff
@@ -22,8 +22,9 @@
     "react-component"
   ],
   "scripts": {
-    "build": "npm run build:icons && npm run build:babel && npm run build:copy-files",
+    "build": "npm run build:icons && npm run build:babel && npm run build:babel:es2015module && npm run build:copy-files",
     "build:babel": "babel --presets es2015,react -d ./build ./src",
+    "build:babel:es2015module": "BABEL_ENV=modules babel ./src/index.js --presets=../../../scripts/material-ui-babel-preset-es2015 --out-file ./build/index.es.js",
```

Comment:
Some naive question as I haven't tried.
Would copy pasting the build commands for the main package work?
https://github.com/callemall/material-ui/blob/next/package.json#L26-L27